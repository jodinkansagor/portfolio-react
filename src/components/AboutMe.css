.headshot {
  img {
    width: 170px;
    border-radius: 100%;
  }

  display: flex;
  flex-direction: column;
  align-items: center;
  margin-top: 30px;

  p {
    max-width: 70%;
  }

  @media (min-width: 800px) {
    p {
      font-size: 20px;
    }
  }

  @media (min-width: 1000px) {
    flex-direction: row;

    img {
      margin-left: 45px;
      margin-right: 20px;
    }

   
  }
}

