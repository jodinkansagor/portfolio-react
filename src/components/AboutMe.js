import React from 'react';
import styles from '../components/AboutMe.css';

const AboutMe = () => {
  return (
    <section className={styles.headshot}>
      <img src="../src/assets/headshot.jpg" />
      <p>I am a full-stack software developer in Portland, OR. My diverse background working with large data-sets, creative problem solving, and managing diverse communities has given me the skills to adapt to any environment. Learning to play roller derby as an adult has taught me that I can learn anything that I put my mind to.</p>
    </section>
  );
};

export default AboutMe;
