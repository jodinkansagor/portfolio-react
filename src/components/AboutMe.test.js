import React from 'react';
import { shallow } from 'enzyme';
import AboutMe from './AboutMe';

describe('AboutMe component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<AboutMe />);
    expect(wrapper).toMatchSnapshot();
  });
});