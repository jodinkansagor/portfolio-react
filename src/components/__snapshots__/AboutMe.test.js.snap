// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe component matches a snapshot 1`] = `
<section
  className="headshot"
>
  <img
    src="/path/image"
  />
  <p>
    I am a full-stack software developer in Portland, OR. My diverse background working with large data-sets, creative problem solving, and managing diverse communities has given me the skills to adapt to any environment. Learning to play roller derby as an adult has taught me that I can learn anything that I put my mind to.
  </p>
</section>
`;
